packages/web/src/utils/third-party/cli-files/plugins/index-html-webpack-plugin.ts

```diff
@@ -74,13 +74,16 @@ export class IndexHtmlWebpackPlugin extends IndexHtmlGenerator {
 
       try {
         for (const [entryName, entrypoint] of this.compilation.entrypoints) {
-          const entryFiles: FileInfo[] = entrypoint?.getFiles()?.map(
-            (f: string): FileInfo => ({
-              name: entryName,
-              file: f,
-              extension: extname(f),
-            })
-          );
+          const entryFiles: FileInfo[] = entrypoint
+            ?.getFiles()
+            ?.filter((f) => !f.endsWith('.hot-update.js'))
```

Comment:
This is needed  to avoid hot-update files from being included in generated HTML (which breaks).